.k8/helm3/ssv-liquidator/templates/liquidator-j1-1.yml

```diff
@@ -2,40 +2,40 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: ssv-liquidator-worker-svc
+  name: liquidator-j1-1-svc
   namespace: {{ .Values.namespace }}
   labels:
-    name: ssv-liquidator-worker-svc
+    name: liquidator-j1-1-svc
```

Comment:
@systemblox lets have the same name convention as ssv nodes for example -prater-jato-v1-1